bespoke-phpcbf

```diff
@@ -1,6 +1,11 @@
 #!/usr/bin/env php
 <?php
 
+if (count($_SERVER['argv']) >1) {
```

Comment:
What is the parameter that is passed that is not supported? :D 

This message should say that no parameters are allowed to be passed.

Reply:
Right, I interpreted my words as many all aren't will tweak.